chapter_3/automation_using_ansible.md

```diff
@@ -1,11 +1,28 @@
 # Automation Using Ansible
+## Introduction to Ansible
+Ansible is an opensource tool which serves for configuration management, orchestration, provisioning and deployments. It is developed using python and runs on Windows, Mac, and UNIX-like systems. The Ansible itself is agentless i.e you don't need to install any client side software.It uses SSH connections. Hence,if you have well-oiled SSH setup, then you are roll up to use ansible in your environment. This also means that you can install it on only one machine and control your infrastructure from that machine itself.
 
-- Intro to Ansible
-- ManageIQ and Ansible
-- Ansible Terms
-  - Modules
-  - Playbooks
-  - Roles
-  - Inventory
-  - Job and Job Template
-- Ruby and Ansible
+### ManageIQ and Ansible
+
+Before ease of Ansible reached out to us, one was expected to be quite comfortable with Ruby scripting language and [Automate object model](https://pemcg.gitbooks.io/mastering-automation-in-cloudforms-4-2-and-manage/content/introduction_to_the_automate_datastore/chapter.html) to develop his own automation scripts. But, Now ManageIQ do provide varios provisioning and service management processes. Thus, after Fine release, we can use Ansible playbooks for automation.
+
+#### Ansible Terms
```

Comment:
This one should have h3(###).
And all the terms under this, should have h4(####)